### PR TITLE
get-swap-timeout functionality

### PR DESCRIPTION
**Pull request by phazejeff**
Added get-swap-timeout subcommand to swapcli as described in issue #177

### CODE REVIEW

**Review by noot — COMMENTED** [review]
nice, looks good to me! could you also update `docs/rpc.md`? feel free to base the description off the existing `personal_setSwapTimeout` one. after that this will be good to merge :)

**Comment by phazejeff** [comment]
`personal_setSwapTimeout` actually wasn't in the `docs/rpc.md` doc, I went ahead and added both set and get swapTimeout to the document

File: docs/rpc.md [hunk 1]
```diff
@@ -156,6 +156,38 @@ curl -X POST http://127.0.0.1:5002 -d '{"jsonrpc":"2.0","id":"0","method":"perso
...
+
+### `personal_getSwapTimeout`
+
+Returns the duration between swap initiation and t0 and t0 and t1, in seconds
+
+Parameters:
+- none
+
+Returns:
+- `timeout`: timeout in seconds, default is 0
```
Comment (noot) [inline docs/rpc.md:183]:
```suggestion
- `timeout`: timeout in seconds
```

same here

**Comment by phazejeff** [comment]
Just made those changes, but get-swap-timeout does seem to return 0 without any changes, at least for me

**Review by noot — CHANGES_REQUESTED** [review]
nice work, thanks for the contribution!

**Review by noot — APPROVED** [review]
sorry meant to approve :p

**Comment by phazejeff** [comment]
thats weird, it was passing all the checks before... all I did was edit the rpc.md file lol?

**Comment by noot** [comment]
@phazejeff no, it was an unrelated unit test that had an issue! I'll merge master which should fix it

**Pull request merged**